akka-cluster/src/main/scala/akka/cluster/Member.scala

```diff
@@ -243,18 +245,39 @@ object MemberStatus {
       Removed → Set.empty[MemberStatus])
 }
 
+object UniqueAddress extends AbstractFunction2[Address, Int, UniqueAddress] {
+
+  // for binary compatibility
+  @deprecated("Use Long UID apply instead")
```

Comment:
Didn't know what version number to put, 2.4-ARTERY-RC1 won't make sense to users, should I just put 2.4.11?